tpv13/tpv13.ipynb

```diff
@@ -292,7 +292,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "docker cp my_docker:/home/training/tpv13/my_output ."
+    "# Execute on your local machine, not within this notebok!", 
+    "\n",
+    "export my_docker=!(docker ps --format \"{{.Names}}\")",
```

Comment:
remove !
?

Reply:
Has to be a `$` to store output in a bash variable, thanks for noticing!